Rocket.Chat/Extensions/URLExtension.swift

```diff
@@ -81,4 +81,14 @@ extension URL {
         return newURL
     }
 
+    mutating func removingDuplicatedSlashes() -> URL {
```

Comment:
Can you write tests for this @rafaelks? Does it work with https:// ?

Reply:
Done! :-)

Reply:
Sorry for not catching this before, but does this method need to be mutating? If it must be mutating then we should rename it to "removeDuplicatedSlashes".